app/controllers/service_provider_controller.rb

```diff
@@ -0,0 +1,15 @@
+class ServiceProviderController < ApplicationController
+  protect_from_forgery with: :null_session
+
+  def update
+    # TODO: shared secret token to limit authz to corresponding env dashboard
+
+    if FeatureManagement.use_dashboard_service_providers?
+      ServiceProviderUpdater.new.run
+      SecureHeadersWhitelister.new.run
+    end
+
+    # always return success in case feature is off
+    render json: { status: 'thanks' }
```

Comment:
No human being should ever see the response, since it used only by https://github.com/18F/identity-dashboard/blob/master/app/services/service_provider_updater.rb

So to whom should the message be helpful? Other developers?